tests/user_commands/tests.py

```diff
@@ -331,9 +331,9 @@ def test_no_existent_external_program(self):
 
     def test_get_random_secret_key(self):
         key = get_random_secret_key()
-        self.assertEqual(len(key), 50)
+        self.assertEqual(len(key), 63)
```

Comment:
Maybe add a comment here explaining why it's `50 + 13`. Overall it looks good to me!